docs/source/using/search.md

```diff
@@ -180,6 +180,7 @@ When iterating over metrics, use the `metric` keyword which represents the track
 | -------- | ----------- |
 | `name` | Metric name |
 | `context` | Metric context dictionary |
+| `values.[OPTION]` | Metric aggregated data about metric values such as `last`, `last_step` and etc. |  
```

Comment:
should it be `metric.values.last_step` or `metric.steps.last`?

Reply:
and if the only supported option currently is `last`, then probably it is worth considering putting just `values.last`